src/Tools/ProcessWatchdog/ConsoleUtils.cs

```diff
@@ -13,14 +13,20 @@ internal static void LogMessage(string format, params object[] args)
                 string.Format(CultureInfo.CurrentCulture, format, args));
         }
 
-        internal static void LogError(string messageFormat, params object[] args)
+        internal static void LogError(ErrorCode errorCode, string messageFormat, params object[] args)
         {
```

Comment:
Per @davkean, modified ProcessWatchdog so that it produces error messages like this: "PW000n: Some message"

This will make it easy to write a Jenkins "failure cause" to scrape ProcessWatchdog messages out of the log.